README.md

```diff
@@ -612,7 +612,7 @@ const listResults: drive_v3.Schema$FileList = res.data;
 ```
 
 ### HTTP/2
-This library has **experimental** support for [HTTP/2](https://developers.google.com/web/fundamentals/performance/http2). This support is experimental, and should not be used in production.  To enable it for testing purposes, use the `http2` option anywhere request parameters are accepted:
+This library has support for [HTTP/2](https://developers.google.com/web/fundamentals/performance/http2). This support is well-tested and suitable for use in production.  To enable it, use the `http2` option anywhere request parameters are accepted:
```

Comment:
```suggestion
This library has support for [HTTP/2](https://developers.google.com/web/fundamentals/performance/http2). To enable it, use the `http2` option anywhere request parameters are accepted:
```

Unless we have the experimental language, I think it's implied that the behavior is suitable for production :+1:

Reply:
Oops - looks like the wrong link was referenced. I intended to link to [this page](https://github.com/googleapis/nodejs-googleapis-common/pull/401) on which @JustinBeckwith remarked: "The API is now over a year old, and we've had no complaints on the surface. There are many tests, and it seems to work great."

I'm obviously very far removed from the development of the APIs; you all are the experts! I inferred from Justin's comment that this API was effectively "stable" for production use. Was that an incorrect assumption?

Reply:
P.S. The actual commit: ["fix: mark the HTTP/2 support as stable"](https://github.com/googleapis/nodejs-googleapis-common/pull/401/commits/8a2466ffb6a19610bf0342afc16cd5f112d52a5f)

Reply:
The API is stable :+1: I just think it's sufficient to remove the language that indicates it's experimental, we don't need to explicitly mention that this feature is usable in production (it should be assumed that any features are stable, unless marked otherwise :+1:).

Reply:
Agreed!